.gitignore

```diff
@@ -3,3 +3,4 @@ roles/**/.linted
 tests/integration/inventory
 tests/output
 __pycache__
+.ansible/
```

Comment:
Why is this needed?

Reply:
running linter locally creates this folder - this to prevent accidentally pushing it to the repo